### ISSUE TITLE
The BOX(P, R==1 map50 and MAP50-95 of yolov8m and s in the training process are also high, ==0.995

### ISSUE THREAD

**Issue by ALIOSKUPER** [ultralytics/ultralytics#14955]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

In the process of using the model to fine-tune the parameters and then train, we found that the situation as the title indicates, below are the main changes to the parameters section
Optimizer: adamw
dropout: 0.2
save_hybrid= true
agnostic_nms= true
Nms =true
Scale= 0.5
Flipud = 0.5
Fliplr = 0.5
Hope to get your help

**Comment by ALIOSKUPER** [comment]
Through comparative experiments, it is finally found that save_hybrid will make the index abnormal when it is true. I hope it can help you to fix it

**Comment by Skillnoob** [comment]
`save_hybrid` should already output a warning when used.
As per [docs](https://docs.ultralytics.com/modes/val/#arguments-for-yolo-model-validation):
> If True, saves a hybrid version of labels that combines original annotations with additional model predictions.

**Comment by ALIOSKUPER** [comment]
> `save_hybrid` should already output a warning when used. As per [docs](https://docs.ultralytics.com/modes/val/#arguments-for-yolo-model-validation):
> 
> > If True, saves a hybrid version of labels that combines original annotations with additional model predictions.

Thank you very much for your reply !!!!! Your reply let me understand the meaning of parameters correctly!!!!：） This happened because I misunderstood the meaning of this parameter

**Comment by pderrenger** [comment]
I'm glad the explanation helped clarify the parameter's function for you. If you encounter any further issues or have additional questions, please feel free to ask. Always ensure you're using the latest version of the package to benefit from the most recent fixes and updates.
